### ISSUE TITLE
yuvadm streams

### ISSUE THREAD

**Issue by guaka** [guaka/radio#47]
plug in @yuvadm's https://github.com/yuvadm/streams

ideally we'd have a generic way to fetch and parse such pages

**Comment by yuvadm** [comment]
:+1: this should turn into a community effort to easily aggregate direct stream URLs

**Comment by guaka** [comment]
how's your coffeescript? :)

**Comment by yuvadm** [comment]
"shalom shalom" :) But I think i can handle it

**Comment by guaka** [comment]
#brainstorm

- It could be nice to turn https://github.com/guaka/radio-meteor/blob/e7ba453308ec1f5dda79b4e46c57f7a24c235793/meteor/channels.coffee into a similar file to yours, at github.com/guaka/streams. My file could be default, and radio.meteor.com/yuvadm/ will fetch @yuvadm streams?
- Could it be useful to use something like `.litcoffee` to format the streams. Or is that not needed?

**Comment by yuvadm** [comment]
`.litcoffee` looks awesome, never heard of it before. If I convert my repo to that, will you be able to pull that data and parse it automatically?

**Comment by guaka** [comment]
Yeah! Excellent, if you can get the file both human readable and machine parseable.

Not yet sure how Radio Meteor should pull in the data, but we'll figure that out :)

**Comment by guaka** [comment]
Turned the channel list into https://github.com/guaka/radio-meteor/blob/master/channels.coffee.md - slowly improving this.

Some brainstorming ideas for continuing:
- everyone can fork this repo (and send pull request for good stuff)
- git submodules? (not my favorite technology)
- fetch and parse file client side [raw](https://raw.githubusercontent.com/guaka/radio-meteor/master/channels.coffee.md)
